tutor/test_t2_18_GuideMonitorSupportAndTrainUsers.py

```diff
@@ -34,15 +34,14 @@
     ])
 )
 
-
 @PastaDecorator.on_platforms(BROWSERS)
 class TestGuideMonitorSupportAndTrainUsers(unittest.TestCase):
     """T2.18 - Guide, Monitor, Support, and Train Users."""
 
     def setUp(self):
         """Pretest settings."""
-        self.ps = PastaSauce()
-        self.desired_capabilities['name'] = self.id()
+        self.ps=PastaSauce()
+        self.desired_capabilities['name']=self.id()
         if not LOCAL_RUN:
```

Comment:
both of these are variable assignment statements so there should be spacing on either side of the '='; parameter assignment excludes the spaces (syntax requirement)